python/python/tests/historical_datasets/README.md

```diff
@@ -0,0 +1,4 @@
+# Historical Datasets
+
+This folder cotnains datasets from older versions of Lance that we use for backwards
```

Comment:
```suggestion
This folder contains datasets from older versions of Lance that we use for backwards
```

Reply:
maybe we could introduce spelling check for our repo? such as https://github.com/crate-ci/typos